spec-main/extensions-spec.ts

```diff
@@ -138,6 +138,45 @@ ifdescribe(process.electronBinding('features').isExtensionsEnabled())('chrome ex
       }
     })
   })
+
+  describe('devtools extensions', () => {
+    let showPanelTimeoutId: any = null
+    afterEach(() => {
+      if (showPanelTimeoutId) clearTimeout(showPanelTimeoutId)
+    })
+    const showLastDevToolsPanel = (w: BrowserWindow) => {
+      w.webContents.once('devtools-opened', () => {
+        const show = () => {
+          if (w == null || w.isDestroyed()) return
+          const { devToolsWebContents } = w as unknown as { devToolsWebContents: WebContents | undefined }
+          if (devToolsWebContents == null || devToolsWebContents.isDestroyed()) {
+            return
+          }
+
+          const showLastPanel = () => {
+            // this is executed in the devtools context, where UI is a global
+            const { UI } = (window as any)
+            const lastPanelId = UI.inspectorView._tabbedPane._tabs.peekLast().id
+            UI.inspectorView.showPanel(lastPanelId)
+          }
+          devToolsWebContents.executeJavaScript(`(${showLastPanel})()`, false).then(() => {
+            showPanelTimeoutId = setTimeout(show, 100)
+          })
+        }
+        showPanelTimeoutId = setTimeout(show, 100)
+      })
+    }
+
+    it('loads a devtools extension', async () => {
+      const customSession = session.fromPartition(`persist:${require('uuid').v4()}`);
+      (customSession as any).loadChromeExtension(path.join(fixtures, 'extensions', 'devtools-extension'))
```

Comment:
This needs to be updated to match the changes made in #21587
```suggestion
      (customSession as any).loadExtension(path.join(fixtures, 'extensions', 'devtools-extension'))
```